lambdas/section-manager-lambda/src/jwt.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  generateBearerToken,
+  generateJwt,
+  getLambdaPrivateJwtKey,
+  JwtConfig,
+} from 'lambda-common';
+
+import config from './config';
+
+/**
+ * calls a few common functions to procure a bearer token for including in
+ * queries to the graph
+ *
+ * @returns Promise that resolves a string
+ */
+export const getJwtBearerToken = async (): Promise<string> => {
```

Comment:
thinking if this function should also be part of the `lambda-common` package & imported by both lambdas?

Reply:
ah, i meant to comment in this file. so yeah, it's duplicated in each lambda, but the reason for that is so we are able to mock JWT functionality in tests.

if this function were exported from `lambda-common`, jest wouldn't be able to mock it (at least not without jumping through some uncomfortable hoops). so basically this function exists as it is as both a convenience for the lambda and a convenience for testing. i'll add a comment to both files so that's more obvious 😄

Reply:
makes sense, ty for adding the comment!